apps/api/drizzle/0012_quiet_exiles.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX IF EXISTS "auto_top_up_enabled_id_idx";--> statement-breakpoint
```

Comment:
question(blocking): It could be done in the same migration above?

Reply:
It could, why is it blocking? It's totally fine this way too imo

Reply:
It's not really blocking, it does work for sure, I just think for a single pr schema change doing just one migration makes more sense.

Reply:
well, for databases which does not support Transactions for DDL changes, it's actually critical to have 1 migration per transactional statement, otherwise you will need to do migration manually if it fails somewhere in the middle.

BUT Postgres actually supports transactions for DDL changes and drizzle-kitt automatically wraps it in transaction. So, having multiple DDL changes in single file is OK.

To solve the issue of conflicts in file names, we need to use timestamp instead of sequential prefix -> <img width="432" alt="Screenshot 2025-02-07 at 04 40 58" src="https://github.com/user-attachments/assets/8b63e6b3-59dd-42bd-9467-7d4aadcb9088" />

Having timestamps in prefix, we will not have collisions and will not actually care about how many files are used for migration